### ISSUE TITLE
OpenNMT: training speed using target token/s instead of source token/s

### ISSUE THREAD

**Issue by pltrdy** [pytorch/examples#75]
Hi,

I just realized that `train.py` is printing speed using target token per second (cf. [train.py#184](https://github.com/pytorch/examples/blob/master/OpenNMT/train.py#L184)).

It turns out that, for the same process, OpenNMT (LUA) is printing source token/s.

This is quite miss-leading for people benchmarking both solutions. 

Note that I the case of text summarization, target token/s and source token/s are way different (5 to 10 times). At first, I saw that, for the exact same task (and same amount of parameters, same batchsize), PyONMT was 9x "slower" than LUA ONMT. 

I'm not sure of the reason why you chose tgt token/s but it would be easier for user to have the same metric I guess. 

Ty for pytorch, ty for pyONMT, very nice work here :)   
`pltrdy`

**Comment by bmccann** [comment]
@pltrdy @srush PyOpenNMT -- nice! I just put up a multi-gpu PR that got merged and has some other fixes if you're going to run translate.py. If I develop further, would you mind if I branched off of PyOpenNMT from now on so that I can coordinate with you more easily? Also, are you in the PyTorch slack?

**Comment by srush** [comment]
Fantastic! I am in pytorch slack, but we also have an opennmt gitter.
https://gitter.im/OpenNMT/openmt , maybe we could use that?

**Comment by bmccann** [comment]
Great! I'll check that out.

**Comment by soumith** [comment]
this was fixed.
Also

OpenNMT has been moved to https://github.com/OpenNMT/OpenNMT-py and will be continuing development there by the OpenNMT team and other community members.

Please send pull requests and issues over there.

Thank you.
